common/events.h

```diff
@@ -91,7 +91,12 @@ enum EventType {
 
 	EVENT_CLIPBOARD_UPDATE = 27,
 
-	EVENT_CUSTOM_BACKEND_HARDWARE = 28
+	EVENT_CUSTOM_BACKEND_HARDWARE = 28,
+
+	EVENT_X1BUTTONDOWN = 29,
```

Comment:
Just reading this enumeration without context, I believe it is hard to understand this is about mouse buttons. Perhaps the naming needs to be adjusted to make it more obvious. Perhaps with a comment explaining X1 and X2 are usually respectively back and forward, to make it easier for backend authors.

Reply:
OK, done.